docs/data/material/experimental-api/css-variables/css-variables.md

```diff
@@ -0,0 +1,128 @@
+# CSS variables
+
+<p class="description">Learn about the experimental API for using CSS variables on Material UI components.</p>
+
+CSS variables is the most upvoted `@mui/system` related issue.
+It allows several improvements in developer experience, like fixing dark mode SSR flickering, allowing multiple themes other than only light and dark, better debugging experience overall, and many others.
+To learn more, check [the RFC](https://github.com/mui/material-ui/issues/27651) we did about it.
+
+A while ago, we added an experimental API to the System package meant to allow styling the components with CSS variables.
+We want now to take it one step further and utilize it in one of the Material UI components.
+
+## Usage
+
+At this moment, the API does not require any breaking changes for the Material UI components, however it depends on using a new experimental provider for the theme, called `Experimental_CssVarsProvider`.
```

Comment:
Based on our experiments it shouldn't require braking changes, I hope it will stay like that in the future (at least we won't implement the bits that would potentially require braking changes till we have v6 alpha), but I really think there shouldn't be breaking changes.